android/app/src/main/java/com/converse/xmtp/Messages.kt

```diff
@@ -238,12 +238,23 @@ suspend fun handleGroupMessage(
     envelope: Envelope,
     remoteMessage: RemoteMessage,
 ): NotificationDataResult {
-    val groups = xmtpClient.conversations.listGroups()
-    val group = groups.find { it.topic == envelope.contentTopic }
+    // No need to sync groups here, because it's already a known group
+    val group = xmtpClient.findGroup(getGroupIdFromTopic(envelope.contentTopic))
```

Comment:
You should have a method for this now
```
xmtpClient.findConversationByTopic(topic)
```

Reply:
@nplasterer It's not on the currently installed SDK version